src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+}
+
+html {
+  --place-holder-color: #3D4E61;
+  --border-color: #E1E7ED;
+
+  font-size: 16px;
+  font-style: normal;
```

Comment:
It's a default value for this 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/91411692/498fa5ff-9ef8-464a-ad4e-754c156966cf)

```suggestion
```